server/utils/IEBrowserDetection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+/**
+ * @description Check if user is using IE browser and if that's the case show browser unsupported messages
+ *
+ * @returns {boolean}
+ */
+export default function IEBrowserDetection() {
+  const scriptString = `
+    var isIE = document.documentMode;
```

Comment:
I think this check might not work in all environments and should be adjusted a bit. One way to detect IE11 is to check if the string `Trident` is in `window.navigator.userAgent` (https://www.w3docs.com/snippets/javascript/how-to-detect-internet-explorer-in-javascript.html).